packages/frontend/src/pages/private/ManageAccountPage.tsx

```diff
@@ -0,0 +1,236 @@
+import {
+  Avatar,
+  Button,
+  Container,
+  Input,
+  Modal,
+  Spacer,
+  Text,
+} from '@nextui-org/react';
+import React, { useEffect, useState } from 'react';
+import { useApi } from '../../hooks/useApi';
+import { useAuth } from '../../hooks/useAuth';
+import { HouseServices } from '../../services/HouseService';
+
+interface ManageAccountPageProps {}
+
+const btnStyle = {
+  padding: '40px',
+};
+
+const ManageAccountPage: React.FC<ManageAccountPageProps> = (
+  props: ManageAccountPageProps,
+) => {
+  const [joinVisible, setJoinVisible] = useState(false);
+  const [createVisible, setCreateVisible] = useState(false);
+  const [houseCode, setHouseCode] = useState('');
+  const [joinedHouse, setJoinedHouse] = useState(false);
+  const [houseData, setHouseData] = useState();
+  const [newPhoneNumber, setNewPhoneNumber] = useState('');
+  const [newAddress, setNewAddress] = useState('');
+  const [newHouseName, setNewHouseName] = useState('');
+  const { user } = useAuth();
+
+  // Hook for the Join Button Modal to be configured
+  const handler = () => setJoinVisible(true);
+  const closeJoinHandler = () => {
+    setJoinVisible(false);
+  };
+
+  // Hook for the Create Button Modal to be configured
+  const createHandler = () => setCreateVisible(true);
+  const closeCreateHandler = () => {
+    setCreateVisible(false);
+  };
+
+  function handleJoiningHouse() {
+    setJoinVisible(false);
+    HouseServices.joinHouse(houseCode).then((data) => {
+      setHouseData(data);
+    });
+  }
+
+  async function handleCreatingHouse() {
+    try {
+      setCreateVisible(false);
+      const newHouse = {
+        address: newAddress,
+        name: newHouseName,
+        phoneNumber: newPhoneNumber,
+      };
+      const response = await HouseServices.createHouse(newHouse);
+    } catch (e) {}
+  }
+
+  // If the user joined house then cannot create house.
+  // use
+  const { data } = useApi('/api/v1/house', {
+    method: 'get',
+  });
+
+  console.log(data);
+
+  useEffect(() => {
+    if (data?.code) {
+      setJoinedHouse(true);
```

Comment:
If a user is in a house they can't create or join a house. Instead simply navigate them to the dashboard.

```ts
const navigate = useNavigate()
navigate("/home")
```

Reply:
Just want to confirm, Do you mean this?
<img width="461" alt="image" src="https://user-images.githubusercontent.com/49899381/158011084-97230337-840d-4dcd-87c4-2236f4978918.png">

Reply:
Yes, but i dont think you need the state if you are navigating them.